back/engines/commercial/bulk_import_ideas/app/views/bulk_import_ideas/web_api/v1/export_form.html.erb

```diff
@@ -139,6 +138,16 @@
           <% elsif field[:format] == :single_line_text %>
             <div class="line"></div>
 
+          <% elsif field[:format] == :ranking %>
+            <div class="options">
+              <% field[:options].each do |option| %>
+                <div class="ranking">
+                  <div class="ranking-box"></div>
+                  <div><%= option[:title] %></div>
+                </div>
+              <% end %>
+            </div>
```

Comment:
Should we use semantic HTML, or is that not important? (No idea how PDF a11y works.)

Reply:
I'm not sure to be honest as it's being printed to PDF by chrome, so I've just used HTML that gets the visuals we need and I think it's just about getting the paper version accessible. We should maybe review the whole thing next week to see if there are any tweaks we need to make.